src/main/resources/io/viash/platforms/nextflow/WorkflowHelper.nf

```diff
@@ -603,10 +606,467 @@ def paramsToList(params, config) {
 }
 
 def paramsToChannel(params, config) {
+  System.err.println("Warning: paramsToChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   Channel.fromList(paramsToList(params, config))
 }
 
 def viashChannel(params, config) {
+  System.err.println("Warning: viashChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
```

Comment:
You're throwing three deprecation warnings when running viashChannel. Can you try ensuring that the deprecation warning only gets printed once? Maybe something like:

```groovy
viashChannelDeprecationWarningPrinted = false

def viashChannel(params, config) {
  if (!viashChannelDeprecationWarningPrinted) {
    viashChannelDeprecationWarningPrinted = true
    System.err.println("Spiderpig, spiderpig!")
  }
  // ...
}
```

Not sure if this will work. Can you try it out?

Reply:
This works!